libdnf/rpm/package_sack.cpp

```diff
@@ -545,9 +545,11 @@ void PackageSack::Impl::load_available_repo(Repo & repo, LoadRepoFlags flags) {
         try {
             // the updateinfo is not a real extension flags = 0
             auto repodata_info = load_repo_ext(
-                repo, SOLV_EXT_UPDATEINFO, Repo::Impl::MD_FILENAME_UPDATEINFO, 0, [](LibsolvRepo * repo, FILE * fp) {
-                    return repo_add_updateinfoxml(repo, fp, 0) == 0;
-                });
+                repo,
+                SOLV_EXT_UPDATEINFO,
+                repo::Repo::Impl::MD_FILENAME_UPDATEINFO,
+                0,
+                [](LibsolvRepo * repo, FILE * fp) { return repo_add_updateinfoxml(repo, fp, 0) == 0; });
```

Comment:
This is not nice but I suggest that it was suggested by clang-format. This is only note